quilt.mod.json/schemas/schema_version_1.json

```diff
@@ -149,36 +149,35 @@
                   "title": "The unless field",
                   "description": "Describes situations where this dependency can be ignored.",
                   "markdownDescription": "Describes situations where this dependency can be ignored. For example:\n```json\n{\n\t\"id\": \"sodium\",\n\t\"unless\": \"indium\"\n}\n```\n\nGame providers and loader plugins can also add their own optional fields to the dependency object for extra context when resolving dependencies. The Minecraft game provider, for instance, might define an \"environment\" field that can be used like so:\n\n```json\n{\n\t\"id\": \"modmenu\",\n\t\"environment\": \"client\"\n}\n```",
-                  "$ref": "#/$defs/dependency_object"
-                },
-                "$ref": "#/$defs/minecraft_game_provider_options"
```

Comment:
Oh no, you shouldn't have removed this; This is effectively matching the expected behavior where loader plugins can extend into dependency objects

Reply:
Ah seems like this is supported in newer versions, so I have to downgrade this

Reply:
Should be resolved now